main.go

```diff
@@ -45,10 +48,12 @@ import (
 )
 
 var (
-	setupLog = ctrllog.Log.WithName("entrypoint")
+	setupLog   = ctrllog.Log.WithName("entrypoint")
```

Comment:
Just fyi we're using `ctrl.Log.WithName("setup")` with `ctrl "sigs.k8s.io/controller-runtime"` here instead. Not sure what difference it makes

Reply:
For the sake ok consistency, changed it :)